okta/table_okta_app_assigned_group.go

```diff
@@ -114,7 +114,7 @@ func listApplicationAssignedGroups(ctx context.Context, d *plugin.QueryData, h *
 func getApplicationAssignedGroup(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
 	logger := plugin.Logger(ctx)
 	logger.Debug("getApplicationAssignedGroup")
```

Comment:
Add app_id qual in list call, also put a filter in list call defination

Reply:
Updated as discussed

Reply:
Added app_id as optional key qual